packages/sentry-plugin/src/sentry-plugin.ts

```diff
@@ -35,15 +33,30 @@ const SentryOptionsProvider = {
  *
  * ## Installation
  *
- * Install this plugin as well as the `@sentry/node` package:
- *
  * ```sh
- * npm install --save \@vendure/sentry-plugin \@sentry/node
+ * npm install --save \@vendure/sentry-plugin
  * ```
  *
  * ## Configuration
  *
- * Before using the plugin, you must configure it with the DSN provided by Sentry:
+ * Setting up the Sentry plugin requires two steps:
+ *
+ * ### Step 1: Preload the Sentry instrument file
+ *
+ * The Sentry SDK must be initialized before your application starts. This is done by preloading
+ * the instrument file when starting your Vendure server:
+ *
+ * ```sh
+ * node --import \@vendure/sentry-plugin/instrument ./dist/index.js
```

Comment:
There is a mismatch between the docs and the package.json, which is exporting the instrument.ts file as `/preload`.